src/app/courses/components/delete-cours-modal/delete-cours-modal.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
There is a small typo in the name of the component.
Also, the best practice is to create a reusable modal window, so the name shouldn't be specific. Just 'modal-window' or 'modal-dialog' would be fine.
Put the component into the core folder after renaming.

Reply:
fixed, thanks